highfidelity/hifi
# [DNM] Marketplace tablet theft prevention

This prevents the tablet from being taken by other avatars, and also allows you to teleport to another domain while the market tablet entity stays with you, as well as being able to create the tablet on domains without the requirement of rez permissions.

## Test plan

- Click the market button in HMD
- Ask another avatar to take or edit the marketplace tablet. (This should be impossible)
- Try teleporting to other domains and notice that the marketplace tablet follows you.
